src/cityreader/cityreader.py

```diff
@@ -21,7 +31,15 @@ def cityreader(cities=[]):
   # Ensure that the lat and lon valuse are all floats
   # For each city record, create a new City instance and add it to the 
   # `cities` list
-    
+    with open('cities.csv') as cityfile:
+      cityreader = csv.reader(cityfile, delimiter=",")
+      line=0
+      for row in cityreader:
+        if line == 0:
+          line+=1
+        else:
+          cities.append(City(row[0], float(row[3]), float(row[4])))
+        # print(cities)
     return cities
```

Comment:
Really nice job here!

One way to skip that first line is to use the built-in Python function, `next()`,

then you can just call `next(cityreader)` to skip the first line before you enter the for-loop to skip the first line.

But, you clearly have a handle on this problem. Nice job!